aspnetcore/web-api/handle-errors.md

```diff
@@ -140,13 +142,101 @@ For web API controllers, MVC responds with a <xref:Microsoft.AspNetCore.Mvc.Vali
 
 ## Client error response
 
-An *error result* is defined as a result with an HTTP status code of 400 or higher. For web API controllers, MVC transforms an error result to a produce a <xref:Microsoft.AspNetCore.Mvc.ProblemDetails>.
+An *error result* is defined as a result with an HTTP status code of 400 or higher. For web API controllers, MVC transforms an error result to produce a <xref:Microsoft.AspNetCore.Mvc.ProblemDetails>.
 
 The error response can be configured in one of the following ways:
 
+1. Use the [problem details service](#pds7)
 1. [Implement ProblemDetailsFactory](#implement-problemdetailsfactory)
 1. [Use ApiBehaviorOptions.ClientErrorMapping](#use-apibehavioroptionsclienterrormapping)
 
+<a name="pds7"></a>
+
+### Problem details service
```

Comment:
We could mention [Disable ProblemDetails response](https://learn.microsoft.com/en-us/aspnet/core/web-api/?view=aspnetcore-7.0#problem-details-for-error-status-codes-1) that will disable `ProblemDetails` generation for API Controllers, including when calling `WriteAsync` for an API Controller endpoint